### PR TITLE
Desktop version.

### PR DESCRIPTION
**Pull request by EmiLoKiJu**
- Fix positions
- Add responses to every size
- Follow Figma design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -147,45 +197,53 @@ <h1 class="aboutmeh1">
...
-            <div>
+            <!-- <div class="form"> -->
                 <h1>
                     I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
                 </h1>
-            </div>
+            <!-- </div> -->
             <form action="https://formspree.io/f/xzbqjzwo" class="formlu" method="post">
                 <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required/>
                 <span class="linespan"></span>
```
Comment (ghost) [inline index.html:249]: - [ ] Great work on your design, But there are  some aspects you can improve on:
 - Kindly reduce the spacing between the form inputs to match the original design.
 - Adjust the padding of around the placeholder text of the `<textarea>` component to match the original design.
 - Adjust the padding around the `get in touch` button to match the original design
 
image

### Checkout the original design

image

**Review by ghost — CHANGES_REQUESTED** [review]
Hi @EmiLoKiJu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -194,35 +252,35 @@ <h1>
                 <textarea id="msg" name="user_message" placeholder="Write me something..." maxlength="500" required></textarea>
                 <button class="submitform" type="submit">Get in touch</button>
```
Comment (V-Blaze) [inline index.html:253]: - [OPTIONAL] You did a great job implementing the contact form and all the changes requested by the previous reviewer. 👍 👍 , I noticed that you did not add interactive states effects to your submit button, like the `Hover` and `Enabled`, Please consider adding this as it greatly improves the user experience of your website. 😄 😸

**Review by V-Blaze — APPROVED** [review]
Hi @EmiLoKiJu , 

# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] Portfolio looks great ✔️
- [ ] Implemented all the design changes suggested by the previous reviewer. ✔️


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**